app/helpers/application_notifications_helper.rb

```diff
@@ -0,0 +1,11 @@
+module ApplicationNotificationsHelper
+  # Translate the event of an activity to the path of a given notification
+  #
+  # @param [Course::Notification, UserNotification] notification whose view path is required
+  # @return [String] the path of a template
```

Comment:
You didn't specify what happens if the object passed in is not of a proper type. Yet, you test it in your spec.